src/pyproject_api/__main__.py

```diff
@@ -0,0 +1,99 @@
+from __future__ import annotations
+
+import argparse
+import os
+import pathlib
+import sys
+import textwrap
+
+from ._frontend import EditableResult, SdistResult, WheelResult
+from ._via_fresh_subprocess import SubprocessFrontend
+
+
+def main_parser() -> argparse.ArgumentParser:
+    parser = argparse.ArgumentParser(
+        description=textwrap.indent(
+            textwrap.dedent(
+                """A minimal PEP-517 frontend.
+
+                This is mainly useful for debugging PEP-517 backends. This frontend will not
+                actually install required build dependencies.
+                """
+            ).strip(),
+            "    ",
+        ),
```

Comment:
I don't want to replicate their desc. Let's replace this with `pyproject.toml file based build frontend`.